R/methods-group-features.R

```diff
@@ -0,0 +1,1049 @@
+#' @title Grouping of LC-MS features
```

Comment:
`Grouping of LC-MS features` is quite generic and overloaded term. /me is wondering if we should dare what we didn't in CAMERA and call 'em compound spectra containing groups of peaks ....

Reply:
Agree, what if we use *Compounding of LC-MS features* as title here?

Reply:
Sounds good to me. Maybe we can run the idea past a few other contributors as well for confirmation.

Reply:
I would not like to rename the `groupFeatures` method as this method (defined in `msFeatures`) is thought to be as generic as possible - it's just a concept of allowing to *group MS features* - in our case here we're trying to group features because they might be from the same compound, but there could also be other use cases - also thinking of proteomics (grouping of peptides to proteins). Thus, I would like to keep the name of the method as it is, but clearly describe the goal of this grouping in `xcms` (i.e. *compounding*).